app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -381,6 +382,15 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
     }
   }
 
+  async stampDocuments() {
```

Comment:
This would be better located in a service. Fetches in components are not best practice.

Reply:
Also, this is now only when the meeting is closed. We want to trigger this for every agenda version